daemon/config_windows.go

```diff
@@ -0,0 +1,26 @@
+package daemon
+
+import (
+	"os"
+
+	flag "github.com/docker/docker/pkg/mflag"
+)
+
+// PlatformSpecific defines the configuration of a docker daemon
+// that is specific to a particular platform.
+type PlatformSpecific struct {
```

Comment:
That's what I was referring to as doing this would change a significant number of files for (IMHO) little gain. Everywhere where config.field is mentioned is going to have to change config.commonconfig.field, hence why I took the lesser of two evils where there are fewer platform specific fields, hence far less files affected.  It also makes more sense IMO that the stuff that's specific be sub-factors (per-se) as opposed to things which are indeed common.